.github/workflows/checks.yml

```diff
@@ -0,0 +1,63 @@
+name: Checks
```

Comment:
nitpick: Please use a name such as "PR checks" so we know at a glance that this is specific the PR flow